packages/next-app/src/app/internalmanager/layout.tsx

```diff
@@ -20,12 +21,12 @@ export default function Layout({ children }: { children: React.ReactNode }) {
           imageSrc={balancerSymbol}
         />
         <InternalManagerProvider>
-          <CheckUnsupportedChain
-            unsupportedChain="Polygon"
-            chainName={chain?.name}
+          <CheckSupportedChains
+            supportedChains={["Ethereum", "Goerli"]}
```

Comment:
I remember we have a `Network` enum somewhere, should instead use it and make any changes if needed because e.g. here we have Ethereum and there IIRC Mainnet.